content/applications/general/voip/axivox/axivox_config.rst

```diff
@@ -0,0 +1,66 @@
+=====================================
+Use VoIP services in Odoo with Axivox
+=====================================
+
+Introduction
+============
+
+Odoo VoIP (Voice over Internet Protocol) can be set up to work together with `Axivox
+<https://www.axivox.com/>`_. In that case, an :doc:`Asterisk server <../asterisk>` is not necessary
+as the infrastructure is hosted and managed by Axivox.
+
+To use this service, `contact Axivox <https://www.axivox.com/contact/>`_ to open an account. Before
+doing so, verify that Axivox covers the company's area and the areas users wish to call.
+
+Configuration
+=============
+
+To configure Axivox on Odoo, first, go to the :menuselection:`Apps` app and search for the `VoIP`
+module. Then install the *VoIP Module*.
+
+Next, go to :menuselection:`Settings --> General Settings --> Integrations`, and fill out the
+:guilabel:`Asterisk (VoIP)` field:
+
+- :guilabel:`PBX Server IP` or :guilabel:`OnSIP Domain`: set the domain created by Axivox for the
+  account (e.g., `yourcompany.axivox.com`)
+- :guilabel:`WebSocket`: type in `wss://pabx.axivox.com:3443`
+- :guilabel:`VoIP Environment`: set as :guilabel:`Production`
+
+.. image:: axivox_config/voip-configuration.png
+   :align: center
+   :alt: Integration of Axivox as VoIP provider in an Odoo database.
+
+.. tip::
+   Access the domain on the Axivox administrative panel. Navigate to `https://manage.axivox.com/
+   <https://manage.axivox.com/>`_. After logging into the portal go to :menuselection:`Users -->
+   Edit (next to any user) --> SIP Identifiers tab --> Domain`.
+
+Configure the VOIP user in the Odoo's user
+------------------------------------------
+
+Next, the user is configured on Odoo. This process must take place for every Axivox/Odoo user using
+VoIP. To start, go to :menuselection:`Settings --> Users & Companies --> Users`, then open the
+user's form to configure :abbr:`VoIP (Voice over Internet Protocol)`. Under the
+:guilabel:`Preferences` tab, fill out the section :guilabel:`PBX Configuration`:
+
+- :guilabel:`VoIP (SIP) Login/Username` / :guilabel:`Browser's Extension Number` = (Axivox)
+  :guilabel:`SIP username`
+- :guilabel:`OnSIP Auth Username` = (Axivox) :guilabel:`SIP username`
+- :guilabel:`SIP Password / VoIP Secret` = (Axivox) :guilabel:`SIP Password`
```

Comment:
Noted, we'll make incremental updates to avoid confusion. This is the only change that needs to be version-specific. I think we merge and then forward port the changes, and I'll go back in and make the changes in another PR to clarify the differences in the version.

Reply:
Leaving unsolved so this is flagged for those changes later.